stdlib/array.gr

```diff
@@ -841,6 +930,9 @@ provide let unzip = array => {
  * @param separator: The separator to insert between items in the string
  * @param items: The input strings
  * @returns The concatenated string
+ * 
+ * @example
+ * Array.join(" ", [> "The", "quick", "brown", "fox"]) // The quick brown fox
```

Comment:
Same As Above:

```suggestion
 * Array.join(" ", [> "The", "quick", "brown", "fox"]) == "The quick brown fox"
```

Reply:
Same comment as before. This is not returning void so simply in line the result.....